package.json

```diff
@@ -81,7 +81,7 @@
     "build:main:prod": "tsc -p tsconfig.build.main.json",
     "build:renderer:prod": "webpack --config webpack/webpack.config.renderer.prod.js",
     "prestart": "npm run build:main",
-    "start": "ZBAY_IS_TESTNET=0 npm run start:renderer",
+    "start": "ZBAY_IS_TESTNET=0 DEBUG='waggle:*,zbay:*' npm run start:renderer",
```

Comment:
Now that I see this, I would guess that ZBAY_IS_TESTNET variable has not been relevant for over a year

Reply:
We'll look at it during refactoring